app/src/main/java/com/hedvig/app/feature/home/ui/HomeFragment.kt

```diff
@@ -181,6 +188,10 @@ class HomeFragment : Fragment(R.layout.home_fragment) {
         .filterNotNull()
         .map { HomeModel.PSA(it) }
 
+    private fun upcomingRenewals(contracts: List<HomeQuery.Contract>) =
+        contracts.filter { it.upcomingRenewal != null }
+            .map { HomeModel.UpcomingRenewal(it) }
```

Comment:
```suggestion
        contracts.mapNotNull { c -> c.upcomingRenewal?.let { HomeModel.UpcomingRenewal(it) } }
```

This is an alternative form you could use, it's by no means required though 🙂